add doc for AlibabaCloud ENI (#15512)
cilium-team/cilium: Documentation/gettingstarted/k8s-install-remove-ack-node.rst

.. only:: not (epub or latex or html)

    WARNING: You are looking at unreleased Cilium documentation.
    Please use the official rendered version released here:
    https://docs.cilium.io

> Cilium will manage ENIs instead of ACK CNI, so either one of listed DaemonSet
> have to be deleted to prevent conflict behavior.

Reviewer: ```suggestion
Cilium will manage ENIs instead of the ACK CNI, so any running DaemonSet from
the list below has to be deleted to prevent conflicts.
```

- ``kube-flannel-ds``
- ``terway``
- ``terway-eni``
- ``terway-eniip``

.. note::

    If you are using ACK with Flannel (DaemonSet ``kube-flannel-ds``),

>     Cloud Controller Manager (CCM) will create route (Pod CIDR) in VPC.
>     If your cluster is an Managed Kubernetes you can not disable this behave.
>     Please consider create a new cluster.

Reviewer: ```suggestion
    the Cloud Controller Manager (CCM) will create route (Pod CIDR) in VPC.
    If your cluster is an Managed Kubernetes you cannot disable this behavior.
    Please consider creating a new cluster.
```

> .. code:: bash

Reviewer: ```suggestion
.. code-block:: shell-session
```
